test/OAuthTest.php

```diff
@@ -154,7 +158,9 @@ public function testOauthClientUsingPostAndGetRequestParametersForSignature()
               ->setTokenSecret('456');
 
         $client = new OAuthClient([
-            'token' => $token
+            'token' => $token,
+            'consumerSecret' => '', // for PHP 8.1 compatibility
```

Comment:
These added keys shouldn't be necessary for PHP 8.1 compatibility: can we somehow  change our sources so this doesn't become a BC issue? 🤔

Reply:
As we decided to add just a string cast instead of changing signature, these keys are no longer required. I removed them